### ISSUE TITLE
[README] Section for Requirements?

### ISSUE THREAD

**Issue by sorianog** [sandoche/Surfable-app#46]
Would be nice to have a dedicated section in the README, that outlines the **Requirements** to run the project.  
When doing a quick search, I wasn't sure what Desktop OS' are supported (i.e MacOS, Windows, Linux).  

I can open up a PR for this quick addition if it makes sense!  

Great work with the project so far!

**Comment by farbodsaraf** [comment]
Yes, it make a lot of sense @sorianog. Your PR is very appreciated.
